app/db/mongodb.py

```
@@ -4,14 +4,7 @@
 from pymongo.database import Database
 from pymongo.errors import PyMongoError
 from app.core.settings import settings
-from app.core.constants import (
-    COLLECTION_SEARCH_HISTORY,
-    COLLECTION_USER_ACTIVITIES,
-    COLLECTION_RECOMMENDATION_INTERACTIONS,
-    TTL_SEARCH_HISTORY_SECONDS,
-    TTL_USER_ACTIVITIES_SECONDS,
-    TTL_RECOMMENDATION_INTERACTIONS_SECONDS,
-)
+from app.core.settings import settings
```

Flake8 (7.3.0)

[error] redefinition of unused 'settings' from line 6

(F811)